build/InstallerInfo.props

```diff
@@ -0,0 +1,10 @@
+<Project>
+  <PropertyGroup>
+    <SdkDebianPackageVersion>$(SdkVersion)</SdkDebianPackageVersion>
```

Comment:
It seems like the Debian properties should probably go together, perhaps in a PG in the debian installer targets file?